services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -107,8 +107,11 @@ def validate_project(app: web.Application, project: Dict):
 
 
 async def delete_project(request: web.Request, project_uuid: str, user_id: str) -> None:
-    await remove_project_interactive_services(user_id, project_uuid, request.app)
-    await delete_project_data(request, project_uuid, user_id)
+    async def remove_services_and_data():
+        await remove_project_interactive_services(user_id, project_uuid, request.app)
+        delete_project_data(request, project_uuid, user_id)
+    ensure_future(remove_services_and_data())
```

Comment:
what about changing the order of the calls? If the database call fails for whatever reason the project will still be in the database but its data will be gone which is kind of inconsistent.

await delete_project_from_db(request, project_uuid, user_id)
ensure_future(remove_services_and_data())